feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SessionPageFragment.kt

```diff
@@ -135,11 +135,14 @@ class SessionPageFragment : DaggerFragment() {
                 }
             }
 
-            childFragmentManager
-                .beginTransaction()
-                .replace(R.id.sessions_sheet, fragment)
-                .disallowAddToBackStack()
-                .commit()
+            // suppress fragment replacement
+            if (childFragmentManager.findFragmentByTag(fragment::javaClass.name) == null) {
```

Comment:
I am worried that Day Fragment's class is the same on Day 1 and Day 2, so when the Fragment is reused by ViewPager, information on another tab will be displayed.
So how about using the SessionPage title as a tag? 👀

Reply:
so cool!! 👍 
I'll do it!

Reply:
And probably we can put Fragment creation in if statement.